app/Libraries/Payments/PaymentProcessor.php

```diff
@@ -341,11 +341,8 @@ protected function getOrder()
 
     /**
      * Convenience method that calls dispatchValidationFailed() and then throws the supplied exception.
-     *
-     * @param Exception $exception
-     * @return void
      */
-    protected function throwValidationFailed(Exception $exception)
+    protected function throwValidationFailed(Exception $exception): void
```

Comment:
this wrapper is kinda useless now with no actual channel? (along with all the event listeners/handlers)

Reply:
Not doing it in this PR, there's going to be a separate cleanup for all this; this PR is only for getting the errors reported again.